pkg/beacon/beacon.go

```diff
@@ -111,7 +128,24 @@ func Initialize(
 			)
 		}
 
+		newEntry := event.NewEntry.Text(16)
```

Comment:
Since we are using `[0x%x]` formatter I think we should be using `events.NewEntry` directly in `Errorf` and `Infof`. In the current setup we are hex-ing twice.

Reply:
done